docs/decisions/2023-12-11_cypress_test_isolation.md

```diff
@@ -0,0 +1,14 @@
+# 2023-12-11 Cypress tests run independently by resetting browser and database state
+
+- design: WRITING-14354
+- status: accepted
+- date: 2023-6-26
+- authors: SupaJoon
+
+## Context and Problem Statement
+
+Cypress tests that mutate browser or Evergreen database state affected outcomes of tests that ran after. This prevented parallelzing tests in CI, running a subset, and thoroughly testing UI flows that mutate Evergreen data.
+
+## Decision Outcome
```

Comment:
It may be nice to add a statement about the takeaways regarding parallelization since that's mentioned as a problem.